Fix parsing of intervals. defined in ms

hc4 opened the pull request:
Fix parsing of intervals. defined in ms.
Increase accuracy for sub-second inrevals.
Closes #5008

hc4 committed:
Fix parsing of intervals. defined in ms

Fix parsing of intervals. defined in ms.
Increase accuracy for sub-second inrevals.
Closes #5008

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=5151) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=5151) before we can accept your contribution.<br/>

torkelo commented:
could you add a unit test for this case in  https://github.com/grafana/grafana/blob/master/public/test/core/utils/kbn_specs.js  ?

hc4 committed:
Tests for sub-ms intervals

Unit test for https://github.com/grafana/grafana/commit/2034d4b9710b0cb9fd0cdf14aa9981cd0417eda4

hc4 commented:
Is there enough tests?
Or should I write more?

torkelo commented:
no that is fine! thank you for contributing to Grafana!
